nextflow.config

```diff
@@ -73,6 +75,7 @@ params {
 // Load base.config by default for all pipelines
 includeConfig 'conf/base.config'
 includeConfig 'conf/codon_slurm.config'
+includeConfig 'subworkflows/ebi-metagenomics/reads_bwamem2_decontamination/nextflow.config'
```

Comment:
This includeConfig 'conf/codon_slurm.config' should be moved inside the ebi {} profile. You don't want to load it for every profile because it will collide with the profile:
test      { includeConfig 'conf/test.config' }
that you will add.
Also, make sure where you want to load includeConfig 'subworkflows/ebi-metagenomics/reads_bwamem2_decontamination/nextflow.config' 
Should it be loaded for every profile? If so, move it in base.config. Only for ebi/codon? Then move it inside codon_slurm.config and so on.